rakelib/labels.rake

```diff
@@ -0,0 +1,80 @@
+require 'octokit'
+
+namespace :labels do
```

Comment:
I wonder if that wouldn't be more logical to move that into the `sync.rake` as a `sync:labels` task?